free-1.md

```diff
@@ -57,12 +164,28 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+The implementer would be able to leverage existing solutions for much of the
+systems aspect. As an example, a prototype of this language could be a compiler
+or transpiler that outputs CSound code, thereby using the existing CSound
+infrastructure for actually generating audio. This leaves plenty of time for
+language decisions to be made.
```

Comment:
It might also depend on whether one decided to have a graphical or text-based syntax. Graphical syntaxes generally require more engineering effort.